src/spurt/io/_slc_stack.py

```diff
@@ -0,0 +1,208 @@
+"""Read test SLC stacks during development."""
+
+from __future__ import annotations
+
+import os
+from collections.abc import Mapping
+from pathlib import Path
+
+import numpy as np
+
+from ._raster import Raster
+from ._three_d import Irreg3DInput
+
+__all__ = [
+    "SLCStackReader",
+]
+
+
+class SLCStackReader:
+    """Read a SLC stack from single files.
+
+    This is a specific type of stack where time domain represents acquisition
+    dates and spatial domain represents SAR acquisition pixels. This class
+    supports two types of stacks
+        - Actual coregistered SLCs, i.e, one raster per date
+        - Phase linked SLCs w.r.t first acquisition which is assumed to have
+        zero phase.
+
+    An additional temporal coherence or a quality file should be provided for
+    pixel selection in the spatial dimension.
+
+
+    Attributes
+    ----------
+    dates: list of str
+        List of acquisition dates in order
+
+    slc_files: dict
+        SLC file corresponding to acquisition date
+
+    temp_coh_file: str
+        Temporal coherence file, indicative of quality
+
+    temp_coh_threshold: float
+        Minimum value of temporal coherence to consider a pixel stable
+
+    raster_shape: tuple of int
+        Shape of rasters in the stack
+
+    """
```

Comment:
Remove extra blank lines here.

```suggestion
    pixel selection in the spatial dimension.

    Attributes
    ----------
    dates: list of str
        List of acquisition dates in order
    slc_files: dict
        SLC file corresponding to acquisition date
    temp_coh_file: str
        Temporal coherence file, indicative of quality
    temp_coh_threshold: float
        Minimum value of temporal coherence to consider a pixel stable
    raster_shape: tuple of int
        Shape of rasters in the stack
    """
```

Reply:
Done